### ISSUE TITLE
Add option to copy to clipboard as SVG (preference)

### ISSUE THREAD

**Issue by RichardScharf** [xournalpp/xournalpp#4916]
**Is your feature request related to a problem? Please describe.**
If I select and copy one or multiple LaTeX formulas to the clipboard the resulting image in my clipboard is not rasterized (which is awesome!)
But including any handwritten element in the selection results in everything being rasterized.

**Describe the solution you'd like**
I would love to see an option to not rasterize handwriting in the clipboard by default. 
(e. g. a checkbox in the preferences panel or a "copy to clipboard as svg" option in the context or edit menu like in Excalidraw)

**Describe alternatives you've considered**
If it is not possible to copy handwriting or shapes to the clipboard as svgs then at least keep the LaTeX formulas vectorized while rasterizing the handwritten parts of the selection only.

**Additional context**
![image](https://github.com/xournalpp/xournalpp/assets/72913455/352bb8ae-7151-48ab-9b6b-af80957be63a)

**Comment by rolandlo** [comment]
Some context is needed: What version of Xournal++ do you use, what OS do you use, what is the version of inkscape you use? Have you tried using the nightly build for Xournal++?

Xournal++ puts things in various formats into the clipboard, including svg and png, regardless of whether it's just a LaTeX formula or if it includes strokes.

![grafik](https://github.com/xournalpp/xournalpp/assets/40485433/a997f369-3984-44cd-ac4d-7b4eb219cd80)

![grafik](https://github.com/xournalpp/xournalpp/assets/40485433/10a3b652-c1e3-4cd7-8f13-4fb56f1d2a68)

**Comment by RichardScharf** [comment]
- OS: Linux Mint 20.3 x86_64
- Version of Xournal++: 1.1.3+dev
- Version of Inkscape: 1.2.2

**Comment by RichardScharf** [comment]
It seems like strokes drawn using the mouse are copied to clipboard as .svg whereas strokes drawn using a pressure sensitive pen tablet are copied to clipboard as .png

**Comment by rolandlo** [comment]
@RichardScharf On Linux Mint you're probably in a x11 session. You can probably use [CopyQ](https://github.com/hluk/CopyQ) to display the various formats in which clipboard content is stored. As I told you, Xournal++ should put strokes in both image/svg and image/png format (and some other formats) into the clipboard at the same time.

**Comment by RichardScharf** [comment]
You are correct! There seems to be an image/svg format in my clipboard as well.
![image](https://github.com/xournalpp/xournalpp/assets/72913455/59191da6-7104-48a2-b4b2-cba138a8d503)
Although copying as svg from excalidraw seems to result in just a text/plain format it results in a correct .svg image upon pasting into Inkscape whereas the image/svg or the image/svg+xml format copied from Xournal++ just pastes a rasterized version of the image.

The reason seems to be that when copying from Xournal++ the resulting svg just contains an `xlink:href="data:image/png;base64` inside it which implies that the whole selection got rasterized at some point in the process.

![image](https://github.com/xournalpp/xournalpp/assets/72913455/f1138ce8-8b0c-4399-8938-d96476408c18)
